src/preset.ts

```diff
@@ -1244,7 +1244,7 @@ export function configureArgs(preset: ConfigurePreset): string[] {
     }
 
     preset.warnings.uninitialized && result.push('--warn-uninitialized');
-    preset.warnings.unusedCli && result.push('--no-warn-unused-cli');
+    !preset.warnings.unusedCli && result.push('--no-warn-unused-cli');
```

Comment:
What is the default if it's undefined?  Should we just check if it's `=== false`?

Reply:
The [manual](https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html) seems to imply we only add this switch if it's equal to false

Reply:
That's a good point. Will also fix other cases.